custom_components/ha_washdata/ws_api.py

```
@@ -5405,6 +5458,186 @@ async def ws_get_dtw_debug(
+                def _run_ml_sugg() -> dict[str, Any]:
+                    from .suggestion_engine import MLSuggestionEngine as _ML  # pylint: disable=import-outside-toplevel
+                    raw = _ML(classic_engine).generate_ml_suggestions() or {}
+                    out: dict[str, Any] = {}
+                    for k in _pg_setting_keys:
+                        entry = raw.get(k)
+                        if isinstance(entry, dict) and entry.get("value") is not None:
+                            v = entry["value"]
+                            try:
+                                out[k] = int(float(v)) if k in _pg_int_keys else round(float(v), 4)
+                            except (TypeError, ValueError):
+                                pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(TypeError, ValueError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(TypeError, ValueError): ...`

(SIM105)